sdks/go/pkg/beam/core/runtime/exec/pardo.go

```diff
@@ -167,6 +177,7 @@ func (n *ParDo) FinishBundle(ctx context.Context) error {
 	return nil
 }
 
+// Down tearsdown DoFn resources.
```

Comment:
"tearsdown" should probably be two words.

Reply:
Done.